nilearn/masking.py

```diff
@@ -518,13 +518,11 @@ def compute_background_mask(
 def compute_multi_background_mask(
     data_imgs,
     border_size=2,
-    upper_cutoff=0.85,
```

Comment:
To check: removing those may cause problems for functions that return "masking strategies" that are expected to have similar APIs. It may be necessary to either keep them or add a `**kwargs` but in either case the doc string should explain why those are there.

Reply:
I double checked and this should not be an issue.